README.md

```diff
@@ -2,3 +2,32 @@ libportal - Flatpak portal library
 ==================================
 
 libportal provides GIO-style async APIs for most Flatpak portals.
+
+
+## Getting started
+
+### The Meson build-system
+
+`libportal` uses Meson to build its projects. To install Meson on your system, follow instructions here: https://mesonbuild.com/Getting-meson.html. If your system is missing a dependency, Meson will tell you which one. How you install your missing dependencies is up to you.
+
+### Build libportal
+```
+meson build/libportal
+ninja -C build/libportal
+```
+
+### Build and run portal-test
+```
+meson build/potal-test -Dbuild-portal-test=true
+ninja -C build/portal-test
+chmod +x ./build/potal-test/portal-test/portal-test
```

Comment:
That shouldn't be needed, it's a compiled executable, it should already be executable.